src/tso/importer/dataImporter.py

```diff
@@ -21,7 +18,8 @@ def getObservations():
 
         observations =[]
         for line in lines:
-            singleObservation = ObservationBlock(line[0],line[1],line[2],line[3],line[4],line[5])
+            print(line)
+            singleObservation = ObservationRequest(line[0],(line[1],line[2]),line[3],line[4],line[5], line[6])
```

Comment:
`(line[1],line[2]` I don't think this is meant to be wrapped in a tuple. I'll fix it in my next merge from the `dataimport` branch

Reply:
WAIT! This is correct, I think it might not be in the `GetObsWithConstraints` my bad